distribution/kernel/carbon-home/repository/conf/carbon.xml

```diff
@@ -735,4 +735,6 @@
     <!-- Configure default management ui path for server -->
     <DefaultManagementUIPath>carbon</DefaultManagementUIPath>
 
+    <!-- Configure legacy authorization runtime -->
```

Comment:
```suggestion
    <!-- Configure legacy identity server authorization runtime -->
```

shall we say identity server here? @AnuradhaSK

Reply:
Fixed